src/lib/dracut/modules.d/30gcp-udev-rules/module-setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+# Install 65-gce-disk-naming.rules and
+# google_nvme_id into the initramfs
+
+# called by dracut
+install() {
+  inst_simple /usr/lib/udev/google_nvme_id
```

Comment:
https://github.com/GoogleCloudPlatform/guest-configs/blob/master/src/lib/udev/google_nvme_id#L20
This script depends on the nvme cli ... which would also have to be included in the dracut module.

Reply:
It's probably worth listing all non-coreutils dependencies to be safe. So it would be something like:

```sh
inst_multiple nvme grep sed
```